packages/paste-website/src/pages/components/toast/index.mdx

```diff
@@ -487,9 +488,10 @@ If you choose to handle state entirely yourself, you can use `<Toaster />` on it
 
 #### ToastContainer Props
 
-| Prop     | Type        | Description | Default |
-| -------- | ----------- | ----------- | ------- |
-| children | `ReactNode` |             | null    |
+| Prop               | Type        | Description                                     | Default |
+| ------------------ | ----------- | ----------------------------------------------- | ------- |
+| children           | `ReactNode` |                                                 | null    |
+| \_\_console_patch? | `boolean`   | A temporary z-index fix for the Console product | null    |
```

Comment:
Yay for documenting our changes! 
Would __console_patch be a prop of Toaster or ToasterContainer (or both?). What's the difference between these two?

Reply:
Both is good. Toaster is the "rite fancy" version which comes with some accessibility and animation and other jazz. 

ToastContainer is the no frills, doesn't do anything apart from shove it in the top corner, predecessor